src/streams/CreateStreamScreen.js

```diff
@@ -8,8 +8,10 @@ import * as NavigationService from '../nav/NavigationService';
 import { useSelector } from '../react-redux';
 import { navigateBack } from '../actions';
 import { getAuth, getOwnEmail } from '../selectors';
+import { getStreamsByName } from '../subscriptions/subscriptionSelectors';
```

Comment:
```
streams: Show alert when stream name already exists.

Show Alert when when we try to create a stream with an existing name.

Fixes: https://github.com/zulip/zulip-mobile/issues/5199
```

Commit message nit: Let's link to the discussion where we decided on this approach; perhaps here: https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/.23M5199/near/1363029 .

There were a few things that suggested this approach instead of others (one being the awkwardness of the server API); it could be helpful context in case we look into changing the design in the future.